arches/app/media/js/viewmodels/node-value-select.js

```diff
@@ -121,6 +121,12 @@ define([
             initSelection: function(element, callback) {
                 var id = $(element).val();
                 var tiles = self.tiles();
+                
+                // fixes #10027 where inputted values will be reset after navigating away  
+                if (self.value()) {
+                    id = self.value()
```

Comment:
And elsewhere in this PR, if you don't mind please add trailing semicolons, thanks!
```suggestion
                    id = self.value();
```

Reply:
Added in [2e569c0](https://github.com/archesproject/arches/pull/10642/commits/2e569c0896444644c3594f093394578be5362d62)